backend/src/routes/emailRouter.js

```diff
@@ -1,13 +1,32 @@
 import express from "express";
+import Email from "../models/email";
```

Comment:
In my experience, leaving out .js extension when importing javascript module caused error (but I don't know why haha). But this will conflict with the ESLint test so we have to wrap this line with /* eslint-disable */ comment so our ESLint test ignores this line. At the end this line will look something like this.

/* eslint-disable */
import Email from "../models/email.js";
/* eslint-disable */